ThePhasesOfCraftship/2_best_practice_first/testingBasics/exercises/2_Arrange_Act_Assert_Backwards/2_2_Stats_Calculator/src/index.ts

```diff
@@ -0,0 +1,41 @@
+export interface IStats {
+  minimumValue: number;
+  maximumValue: number;
+  size: number;
+  averageValue: number;
+}
+
+export default class StatsCalculator {
+  private getExtremes(input: number[]): {
```

Comment:
just my personal preference: all methods that don't use the current instance could be static. In this case all of them